### PR TITLE
chore(starters): add gatsby-starter-sass

### PR DESCRIPTION
**Pull request by colbyfayock**
## Description
Adds a new starter that scaffolds a super minimal Gatsby project with Sass stylesheets

### CODE REVIEW

**Comment by colbyfayock** [comment]
🤔I don't _think_ that failing test is related to my change. i can definitely reproduce on node v12 running the tests locally. here's one:

```
 FAIL  packages/gatsby-plugin-sharp/src/__tests__/process-file.js
  ● Test suite failed to run

    The module '/Users/colby/Code/gatsby/node_modules/sharp/build/Release/sharp.node'
    was compiled against a different Node.js version using
    NODE_MODULE_VERSION 59. This version of Node.js requires
    NODE_MODULE_VERSION 72. Please try re-compiling or re-installing
    the module (for instance, using `npm rebuild` or `npm install`).

      at Runtime.requireModule (node_modules/jest-runtime/build/index.js:353:31)
      at bindings (node_modules/bindings/bindings.js:112:48)
      at Object.<anonymous> (node_modules/sharp/lib/constructor.js:10:34)
```

**Comment by colbyfayock** [comment]
🤔i dont _think_ this would have to do with my change, but i see another PR ahead of mine that isn't failing this specific test...

File: docs/starters.yml [hunk 1]
```diff
@@ -2383,3 +2383,16 @@
     - Uses MDX with Gatsby theme for quick and easy set up
     - Material-ui design with optional config passed into the theme options
     - Gradient background with sitemap, rss feed, and offline capabilities
+- url: https://gatsby-starter-sass.netlify.com/
+  repo: https://github.com/colbyfayock/gatsby-starter-sass
+  description: A Gatsby starter with Sass and no assumptions!
+  tags:
+    - Sass
```
Comment (freiksenet) [inline docs/starters.yml]: There is no `Styles` tag (see https://www.gatsbyjs.org/starters/?v=2 and "categories"). Could you change those to `Styling:Sass`, `Styling:SCSS` and remove `CSS` and `Styles` tags?

Comment (LekoArts) [reply docs/starters.yml]: `Styling:SCSS` is pretty much the only one needed here

Comment (colbyfayock) [reply docs/starters.yml]: didn't occur to me those tags were the categories 🤦‍♂it's fixed now. I would have added `Styling:Sass` too, but that doesnt seem to exist in the current list

**Comment by colbyfayock** [comment]
fixed the conflict in the UI and the linter didn't run, so it didnt pick up the EOL lint issue 😅 

fixed ✅

**Pull request merged**